test/powershell/Modules/Microsoft.PowerShell.Utility/PowerShellData.tests.ps1

```diff
@@ -1,83 +1,63 @@
-﻿Describe "Tests for the Import-PowerShellDataFile cmdlet" -Tags "Feature" {
+﻿Describe "Tests for the Import-PowerShellDataFile cmdlet" -Tags "CI" {
 
     It "Validates error on a missing path" {
 
-        $foundError = ""
         try
         {
             Import-PowerShellDataFile -Path /SomeMissingDirectory -ErrorAction Stop
+            Throw "Execution OK"
```

Comment:
are you suggesting that we track all these down in the test (there are at least high dozens) and change those as well